### ISSUE TITLE
The weird same result with Informer

### ISSUE THREAD

**Issue by HHUYuYang** [thuml/Autoformer#41]
当我使用如下和论文一致的配置训练ETTm2时，取得的结果是 mse:0.08926962316036224, mae:0.22885815799236298
这和论文里的Informer的0.088, 0.225是差不多的，请问是数值有误还是我设置的哪里有问题嘛
(activation='gelu', batch_size=32, c_out=1, checkpoints='./checkpoints/', d_ff=2048, d_layers=1, d_model=512, data='ETTm2', data_path='ETTm2.csv', dec_in=1, des='test', device_ids=[0, 1, 2, 3], devices='0,1,2,3', distil=True, do_predict=False, dropout=0.05, dvices='0,1,2,3', e_layers=2, embed='timeF', enc_in=1, factor=1, features='S', freq='h', gpu=0, is_training=1, itr=1, label_len=48, learning_rate=0.0001, loss='mse', lradj='type1', model='Autoformer', model_id='ETTm2_96_96', moving_avg=25, n_heads=8, num_workers=10, output_attention=False, patience=3, pred_len=96, root_path='./data/ETT/', seq_len=96, target='OT', train_epochs=10, use_amp=False, use_gpu=True, use_multi_gpu=True)
Use GPU: cuda:0

我后来发现是label_len设置的是48而不是96， 这个会对结果有什么影响吗

**Comment by wuhaixu2016** [comment]
（1）单变量数据，短预测长度，模型之间的指标差别本身就很小
（2）关于输入，论文附录中有一些探索，对于周期性强的数据（比如 ETTm2），输入数据变长会引入过多噪声，所以label_len如果过长会造成噪声较多，影响效果。
